src/main/java/jaryl/duke/DataManager.java

```diff
@@ -0,0 +1,101 @@
+package jaryl.duke;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+
+/**
+ * DataManager handles reading/writing data from/to local disk storage
+ */
+public class DataManager {
+    private String filePath;
+    private static final String DELIMITER = " \\| ";
```

Comment:
I like that you used a named constant for your delimiter to avoid the use of magic strings in the code. 👍